Makefile

```
@@ -30,6 +30,9 @@ help:
 .docker-build:
 	make build
 
+.devcontainer-build:
+	make devcontainerbuild
```

I don't think we need a recursive make invocation.
```suggestion
.devcontainer-build: devcontainerbuild
```

---

Oh, I see, we have the same thing for `.docker-build`. Maybe there was a reason for the recursive invocation? I can't think of anything, though.